diff --git a/node_modules/@milkdown/crepe/lib/esm/index.js b/node_modules/@milkdown/crepe/lib/esm/index.js
index d014549..a5da856 100644
--- a/node_modules/@milkdown/crepe/lib/esm/index.js
+++ b/node_modules/@milkdown/crepe/lib/esm/index.js
@@ -2748,7 +2748,8 @@ class MenuView {
     const self = this;
     __privateSet$5(this, _slashProvider, new SlashProvider({
       content: __privateGet$5(this, _content$3),
-      debounce: 20,
+      root: typeof document !== 'undefined' ? document.body : void 0,
+      debounce: 100,
       shouldShow(view2) {
         if (isInCodeBlock(view2.state.selection) || isInList(view2.state.selection))
           return false;
@@ -2757,7 +2758,9 @@ class MenuView {
           (node) => ["paragraph", "heading"].includes(node.type.name)
         );
         if (currentText == null) return false;
-        if (!isSelectionAtEndOfNode(view2.state.selection)) {
+        const activeEl = view2.dom.getRootNode().activeElement;
+        const isMenuFocused = activeEl && __privateGet$5(this, _content$3).contains(activeEl);
+        if (!isSelectionAtEndOfNode(view2.state.selection) && !isMenuFocused) {
           return false;
         }
         const pos = __privateGet$5(self, _programmaticallyPos);
